app/styles/gavant-ember-table/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$table-hover-bg-opaque: #ececec !default;
+$primary: #285a92 !default;
+$white: #fff !default;
+$table-border-color: rgba(#000, 0.075) !default;
+$font-family-base: 'Lato', sans-serif !default;
```

Comment:
I'd probably make this default value a more generic font-family (I think "Lato" is a custom font), maybe take a look at bootstrap's default font family.